src/util.ts

```diff
@@ -6,7 +6,7 @@
 import { randomBytes } from 'crypto';
 
 function bytesToBase64url(s: Buffer): string {
-  return s.toString('base64').replace('+', '-').replace('/', '_');
+  return s.toString('base64').replace(/\+/g, 'Z').replace(/\//g, 'Z');
```

Comment:
I'm mildly concerned with the loss of randomness that this introduces; I remember our security team being very diligent about us using as random a string as possible.  As we discussed, despite the name, there is no real reason the output needs to actually be valid Base64, so it's probably best if we go back to the old replacement characters.  At the very least, the two replacement characters should be different.